src/Config/FileLocatorFactoryInterface.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Config;
+
+use Symfony\Component\Config\FileLocatorInterface;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * Interface for file locator factories.
+ *
+ * @author Leo Feyer <https://github.com/leofeyer>
+ */
+interface FileLocatorFactoryInterface
```

Comment:
Interfaces are for ensuring compatibility between objects I doubt that it makes sense to have a static only interface.
IMO we should either convert the factory to a plain worker class or remove the interface as it will never get passed/used anywhere.